src/components/DocumentUtility/DocumentUtility.tsx

```diff
@@ -47,8 +47,9 @@ export const DocumentUtility: FunctionComponent<DocumentUtilityProps> = ({ docum
               }}
             >
               <ButtonIcon
-                className="bg-white border-2 border-cloud-100 rounded-xl hover:bg-gray-100"
+                className="bg-white border-2 border-cloud-100 rounded-xl hover:bg-cloud-100"
                 aria-label="document-utility-qr-button"
+                style={{ width: "auto", height: "auto" }}
```

Comment:
why need auto?

Reply:
there is a default value of 40px that will me it not center.

Reply:
ic right. able to put `max-width: 100%` to `Printer` and `Download` instead? if cannot then nvm

Reply:
hmmm but it makes no diff leh